torch/distributed/nn/jit/templates/remote_module.py.template

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python3
+from typing import Tuple, Any
+
+import torch
+import torch.distributed.rpc as rpc
+from torch import Tensor, nn
+from torch._jit_internal import Future, RRef
+from ${module_interface_cls_module_name} import ${module_interface_cls_name} as module_interface_cls  # noqa
+
+
+class _RemoteModule(nn.Module):  # noqa
+    module_rref: RRef[module_interface_cls]  # noqa
+    is_scriptable: bool
+    global_unique_name: str
+
+    def __init__(self, module_rref, is_scriptable, global_unique_name):
+        super().__init__()
+        self.module_rref = module_rref
+        self.is_scriptable = is_scriptable
```

Comment:
self.is_scriptable is not used either, do we need this?

Reply:
It's useful for users. As I mentioned, we can't script RemoteModule for users, because scripted modules can't be send over wire. Users will rely on this flag to decide whether to script the remote module returned by this API.